### DISCUSSION TITLE
Problems with ruby and brew in general

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#4843] [Getting started] [answered]
### Output of `brew config`

```shell
==> Please wait... tigers are now brewing
Downloading Tigerbrew's ruby; this may take some time

==> Downloading http://archive.org/download/tigerbrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Already downloaded: /Users/User/Library/Caches/Homebrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
==> Unpacking portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
```

### Output of `brew doctor`

```shell
==> Please wait... tigers are now brewing
Downloading Tigerbrew's ruby; this may take some time

==> Downloading http://archive.org/download/tigerbrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Already downloaded: /Users/User/Library/Caches/Homebrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
==> Unpacking portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
```

### Description of issue

Whenever I open terminal I see
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
(eval):11: unmatched '
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
(eval):6: unmatched '
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
(eval):6: unmatched '
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
(eval):6: unmatched '
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
(eval):6: unmatched 

There are no working homebrew commands
Outputs are always the same:
==> Please wait... tigers are now brewing
Downloading Tigerbrew's ruby; this may take some time

==> Downloading http://archive.org/download/tigerbrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Already downloaded: /Users/User/Library/Caches/Homebrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
==> Unpacking portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.

Currently using 
Mac OS 14.1 Beta and xCode 15 (not beta)

**Comment by SMillerDev** [comment]
You seem to have installed Tigerbrew instead of Homebrew, try installing from brew.sh instead.

**Reply by ghost** [reply]
This isn't the reason. I have used homebrew before and it stopped working for no reason( at least i can't find it). I also tried to reinstall, but it doesn't work 
For example
user $ brew
==> Please wait... tigers are now brewing
Downloading Tigerbrew's ruby; this may take some time

==> Downloading http://archive.org/download/tigerbrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Already downloaded: /Users/user/Library/Caches/Homebrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
==> Unpacking portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.
That is your suggestion 

User$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R iuser:admin /opt/homebrew
==> Downloading and installing Homebrew...
HEAD is now at c0c8a4dd4 Merge pull request #16113 from Bo98/audit-retry
Warning: /opt/homebrew/bin is not in your PATH.
  Instructions on how to configure your shell for Homebrew
  can be found in the 'Next steps' section below.
==> Installation successful!

==> Homebrew has enabled anonymous aggregate formulae and cask analytics.
Read the analytics documentation (and how to opt-out) here:
  https://docs.brew.sh/Analytics
No analytics data has been sent yet (nor will any be during this install run).

==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations

==> Next steps:
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

user$ brew help
==> Please wait... tigers are now brewing
Downloading Tigerbrew's ruby; this may take some time

==> Downloading http://archive.org/download/tigerbrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Already downloaded: /Users/user/Library/Caches/Homebrew/portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
==> Unpacking portable-ruby-2.3.3.tiger_g3.bottle.tar.gz
Error: Failed to install vendor ruby.
Error: Failed to install vendor Ruby.

**Reply by SMillerDev** [reply]
Clearly you have installed tigerbrew and it's first in your path because brew doesn't ever mention tigers. Try running `which -a brew` to see where you're actually running from.

**Reply by ghost** [reply]
User$ which -a brew
/usr/local/bin/brew

**Reply by SMillerDev** [reply]
Yeah, that's not the install you just posted about:
```
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

```

**Reply by ghost** [reply]
So what I need to do?

**Reply by SMillerDev** [reply, marked as answer]
If you add `eval $(/opt/homebrew/bin/brew shellenv)` to your shell's rc file it should add this last install to the path before your tigerbrew install.

**Comment by ghost** [comment]
Thank you so much!
